### DISCUSSION TITLE
CR Point de Synchronisation 05/11/2021

### DISCUSSION THREAD

**Discussion by ACTLEM** [cnumr/best-practices#192] [General]
## CR Point de Synchronisation du Vendredi 05/11/2021

Présents: Florine, Nicole, Brice, Clément, Jean-Philippe, Nicolas, Rémi, Thomas L. 

### Situation

- 20 Discussions non traitées (avec 9 propositions d'ajouts et 6 propositions de suppression)
- 14 Bonne pratiques validées pour ajout
- 13 Bonnes pratiques validées pour modification
- 16 Bonnes pratiques validées pour suppression

### Points abordées:

Discussions:

- Suivi des discussions: Il n'est pas évident de tout suivre les discussions et notamment uniquement celles qui nous intéressent (notification par e-mail ne filtre pas ^^). Les discussions traitées sont marquées comme "Answered", filtrer sur dessus permet de n'avoir que celles encore en cours.
- Discussions validées: Les Issues ont été créées suite aux différentes discussions (seule 1 discussion pour l'instant n'a pas donné suite). L'arbitrage est réalisé par l'équipe projet. Pour l'instant, les discussions faisant plutôt consensus ont toutes été traitées. Il reste désormais surtout des discussions où il y a encore un peu de débat ou un désaccord.
- Discussions faisant débat: Un point visio devra être mis en place par l'initiateur de la discussion ou par un membre de l'équipe projet (proposer un Framadate dans la discussion). Ce point de 15-30mins devra permettre de trouver un accord. A noter que si aucun accord n'est trouvé, l'équipe projet devra arbitrer.
- Fin des discussions: L'ajout de nouvelles discussions est possible jusqu'au 12/11 à 18h00. Autrement dit, toute discussion créée après cette date ne sera pas prise en compte dans la V4. Les discussions en cours devront être traitées également ou très rapidement sous peine de ne pas pouvoir être prises en compte à temps pour la V4.

Prise en charge des Issues:

- Les droits des contributeurs ne sont pas suffisants car le repository est privé. Cela a été réglé de suite après la réunion. Si vous avez des difficultés de droits pour créer une branche, un fichier , n'hésitez pas à contacter Thomas.
- Lors de l'ajout d'une bonne pratique, utiliser le template https://github.com/cnumr/best-practices/blob/main/resources/BP_xxxx_fr.md
- Ne pas directement réaliser de commit sur la branche main (le repository n'étant pas public, elle ne peut pas être protégée), passer par une Pull Request d'autant plus qu'elle pourra être revue et soumise à suggestion d'amélioration par d'autres contributeurs le cas échéant

Mise en oeuvre/Impacts environnementaux:

- Travailler sur le passage de 3 à 5 niveaux après une vision claire des bonnes pratiques qui devront être en cours de finalisation (semaine du 22 au 28 novembre?)
- Participation des contributeurs, chacun peut avoir son expérience sur la mise en oeuvre ou la perception des impacts d'une bonne pratique
- Avoir un système de vote/sondage pour toutes les bonnes pratiques

Autres sujets:

- Deux pages sont proposées à être écrites sur le repository en vue de son ouverture: une sur la démarche d'écoconception (notamment sur le fait que c'est une démarche d'amélioration continue) et une sur l'outillage (pour mesurer les effets de la mise en place de bonnes pratiques)
- Suggestion: Rappeler dans le livre de la V4 ou sur le repository, la philosophie de cette version (rajeunissement, pourquoi des bonnes pratiques ont été supprimées (tests), ...) car celle-ci risque de changer par rapport aux autres versions et peut poser questions sur le terrain

### Actions

- Organiser sur les discussions à débat une visio (en proposant des dates via un framadate) de 15-30 minutes pour prise de décision: Chaque créateur de discussion marqué comme à discuter (ajout d'un flag "visio" et d'une note dans les discussions concernées) - deadline: ~12/11/2021~ 19/11/2021 (décalage d'une semaine pour cause de pont du 11/11 et après vérification vis à vis d'Eyrolles)
- Trouver un outil pour réaliser le sondage/vote sur les niveaux de mise en oeuvre et impacts environnementaux: Rémi, Brice, Thomas - deadline: 19/11/2021
- Donner le bon niveau de droits aux contributeurs: Thomas - deadline: 05/11/2021 (done)

**Comment by devcrafting** [comment]
J'ai refait une passe sur les BP non discutées, ça m'a conforté dans l'idée qu'il faudrait a minima avoir une discussion par BP.

Il y a aussi des sujets pas/peu abordés : ce qui tourne autour de CSS, du DOM, dont je ne suis pas vraiment un bon relecteur...

Sinon il me paraît super ambitieux de clôturer au 12/11 (d'autant que cette semaine est à nouveau amputée qvec un pont, comme la semaine dernière avec le 1er...). J'ai du mal à voir comment on va arriver à caler les points sur les discussions à débat en 3 jours.
Bon après, je me dis que si ce n'est pas pour être version, ce sera une démarche initiée pour une prochaine 😋

**Reply by ACTLEM** [reply]
Salut,

Merci beaucoup. J'ai eu une réponse de Fred par rapport à Eyrolles. C'est jouable de décaler d'une semaine ce qui ferait une fin totale des contributions sur le repository le 04/12 (derniers commits "bruts"). Pour donner de la visibilité, il faut que nous ayons tout finalisé/stabilisé complètement la version (typo, numérotation, impacts, moe, priorité, première mise en forme en fonction du nombre de bp, ...) pour le 18 décembre maximum.
